### PR TITLE
docs: update information about PRO images

### PR DESCRIPTION
**Pull request by renanrodrigo**
After the release, pinging CPC should be sufficient to update the cloud
images.

## Desired commit type
 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist
 - [ ] I have updated or added any unit tests accordingly
 - [ ] I have updated or added any integration tests accordingly
 - [x] I have updated or added any documentation accordingly

### CODE REVIEW

File: RELEASES.md [hunk 1]
```diff
@@ -256,15 +256,6 @@ If this is your first time releasing ubuntu-advantage-tools, you'll need to do t
...
- 2. Check Xenial, Bionic, Focal, Hirsute, Impish packages
- 3. Select Destination PPA: UA Client Premium [~ua-client/ubuntu/staging]
- 4. Select Destination series: The same series
- 5. Copy options: "Copy existing binaries"
- 6. Click Copy packages
- 7. Notify Pro Image creators about expected Premium PPA version (patviafore/powersj)
- 8. Once new PRO AMIs are publicly available run `./tools/refresh-aws-pro-ids` to update AMIs we test during CI runs
+## Cloud Images Update
+
+After the release process is finished, if there is a need to push a new cloud image with the new version of UA, CPC must be informed. They will be responsible to update the images using the package from the pockets it was released to (whether it is the `stable` PPA or the`-updates` pocket).
```
Comment (lucasmoura) [inline RELEASES.md:261]: I think we should always ping CPC with a new release. But now I have a another question regarding the release. When should we upload the package to `stable` ? If we do it after the package hits `updates`, then there is no need anymore for using the package on stable and if we upload it to stable after the package moves to `proposed`, errors can be found on the package during the testing phase invalidating that package as `stable`.

So I don't know if we should keep that ppa anymore. What do you think @CalvoM @renanrodrigo  @blackboxsw @orndorffgrant ?

Comment (renanrodrigo) [reply RELEASES.md]: Updated according to post-standup discussion

**Comment by renanrodrigo** [comment]
Updated according to post-standup discussion:
- Always ping CPC when the release is done
- Keep the Stable PPA but put the packages there as they hit proposed

**Review by lucasmoura — APPROVED** [review]

**Review by orndorffgrant — APPROVED** [review]
LGTM

**Pull request merged**